src/GUI/RewardDialogue.java

```diff
@@ -0,0 +1,173 @@
+package GUI;
```

Comment:
Remove RewardDialogue.java file from this commit (take back up of the file and run command 'git rm src/GUI/RewardDialogue.java'